README.md

```diff
@@ -174,11 +174,11 @@ docker run -p 80:80 kennethreitz/httpbin
 ## Generating Documentation
 
 The Doxygen references were created using Doxygen version 1.8.20. To generate the
-Doxygen pages, please run the following commands:
+Doxygen pages, please use the provided Python script, [docs/doxygen/generate_docs.py](docs/doxygen/generate_docs.py):
 
 ```shell
-cd libraries/standard/coreMQTT
-doxygen docs/doxygen/config.doxyfile
-cd ../../..
-doxygen docs/doxygen/config.doxyfile
-```
\ No newline at end of file
+cd <CSDK_ROOT>
+python3 docs/doxygen/generate_docs.py --root .
```

Comment:
Should submodules be cloned before calling the python script?

Reply:
Yes. But I didn't want to make it part of the script.